services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -141,6 +142,8 @@ protected void doControllerMsg(CommandMessage message) {
 
     private void handleCommand(CommandMessage message, String replyToTopic, Destination replyDestination)
             throws FlowCommandException {
+        logger.debug("Handling message: '{}'. Topic: '{}'. Destination: '{}'.",
```

Comment:
Now we must remove line
```
  <logger name="org.openkilda.floodlight.kafka.RecordHandler" level="DEBUG"/>
```
from FL's logging config.

Reply:
Remove at all or just change log level to info?